src/modules/facility/facility.service.issue-facility-by-identifier.test.ts

```diff
@@ -49,77 +54,11 @@ describe('FacilityService', () => {
+      when(acbsFacilityServiceGetFacilityByIdentifier).calledWith(facilityIdentifier, idToken).mockResolvedValueOnce(mockResponse);
 
+    const expectAcbsUpdateFacilityToBeCalledWith = (acbsUpdateFacilityRequest: AcbsUpdateFacilityRequest) =>
       expect(acbsFacilityServiceUpdateFacilityByIdentifier).toHaveBeenCalledWith(portfolioIdentifier, acbsUpdateFacilityRequest, idToken);
-    });
-
-    it('uses existing facility data to fill missing update request data', async () => {
-      const newFieldData = 'test';
-      const modifiedAcbsGetExistingFacilityResponse = { ...acbsGetExistingFacilityResponse, NewField: newFieldData };
-      const modifiedAcbsUpdateFacilityRequest = { ...acbsUpdateFacilityRequest, NewField: newFieldData };
-
-      when(acbsFacilityServiceGetFacilityByIdentifier).calledWith(facilityIdentifier, idToken).mockResolvedValueOnce(modifiedAcbsGetExistingFacilityResponse);
-
-      await issueFacility(updateFacilityRequest);
-
-      expect(acbsFacilityServiceUpdateFacilityByIdentifier).toHaveBeenCalledWith(portfolioIdentifier, modifiedAcbsUpdateFacilityRequest, idToken);
-    });
-
-    it('removes AdministrativeUserIdentifier from the ACBS update request body', async () => {
-      const modifiedAcbsGetExistingFacilityResponse = {
-        ...acbsGetExistingFacilityResponse,
-        AdministrativeUserIdentifier: valueGenerator.string(),
-      };
-      when(acbsFacilityServiceGetFacilityByIdentifier).calledWith(facilityIdentifier, idToken).mockResolvedValueOnce(modifiedAcbsGetExistingFacilityResponse);
-
-      await issueFacility(updateFacilityRequest);
-
-      expect(acbsUpdateFacilityRequest).not.toHaveProperty('AdministrativeUserIdentifier');
-    });
-
-    // This is existing behaviour from previous implementations of this service and is maintained
-    it('does not deep merge existing facility object data to fill missing update request data', async () => {
-      const newRiskCountryCode = 'ZZZ';
-      const modifiedUpdateFacilityRequest = { ...updateFacilityRequest, riskCountryCode: newRiskCountryCode };
-
-      const newFieldValue = 'New Field';
-      const modifiedAcbsGetExistingFacilityResponse = {
-        ...acbsGetExistingFacilityResponse,
-        RiskCountry: {
-          CountryCode: acbsGetExistingFacilityResponse.RiskCountry.CountryCode,
-          NewField: newFieldValue,
-        },
-      };
-
-      const modifiedAcbsUpdateFacilityRequest = {
-        ...acbsUpdateFacilityRequest,
-        RiskCountry: { CountryCode: newRiskCountryCode },
-      };
-
-      when(acbsFacilityServiceGetFacilityByIdentifier).calledWith(facilityIdentifier, idToken).mockResolvedValueOnce(modifiedAcbsGetExistingFacilityResponse);
-      await issueFacility(modifiedUpdateFacilityRequest);
-
-      expect(acbsFacilityServiceUpdateFacilityByIdentifier).toHaveBeenCalledWith(portfolioIdentifier, modifiedAcbsUpdateFacilityRequest, idToken);
-    });
 
     it('throws an error if facility is unissued', async () => {
```

Comment:
Different ticket I know, but I think this should read
`'throws an error if the new facility stage code is not issued'`
because the current wording makes me think the error is about the existing facility being unissued instead